beta/src/utils/prepareMDX.js

```diff
@@ -12,7 +12,7 @@ export const PREPARE_MDX_CACHE_BREAKER = 2;
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 export function prepareMDX(rawChildren) {
-  const toc = getTableOfContents(rawChildren);
+  const toc = getTableOfContents(rawChildren, /* depth */ 10);
```

Comment:
Bump the cachebreaker above in this file :)

Reply:
TIL thanks!